models/members.js

```diff
@@ -35,6 +35,31 @@ const fetchMembers = async () => {
   }
 }
 
+/**
+ * Fetches usernames of all the members in the database
+ * @return {Promise<userModel|Array>}
+ */
+
+const fetchMemberUsernames = async () => {
```

Comment:
Rather than creating a new function for just getting the usernames, can we use this function https://github.com/Real-Dev-Squad/website-backend/blob/develop/models/members.js#L14 and then filter out usernames from it?